_posts/2018-07-26-save-the-date-world-moveit-day.markdown

```diff
@@ -21,7 +21,8 @@ We will be having several international event locations including:
 - [PickNik Consulting](https://picknik.ai/), Boulder, USA
 - [Fraunhofer IPA](https://www.ipa.fraunhofer.de/en.html) / [RIC-EU](https://rosindustrial.org/ric-eu/), Stuttgart, Germany
 - [ROS-Industrial Asian Pacific Consortium](http://rosindustrial.org/ric-apac/), Singapore
-- [Tokyo Opensource Robotics Kyokai Association (TORK)](http://opensource-robotics.tokyo.jp/), Kashiwa, Japan
+- [Tokyo Opensource Robotics Kyokai Association (TORK)](http://opensource-robotics.tokyo.jp/), Tokyo, Japan (held on Friday, October 26!)
+- [OMRON SINIC X Corporation](http://tinyurl.com/omronsinicx), Kashiwa, Japan
```

Comment:
i tried separating these two - the are distinct events right? one is in Tokyo and one is in Kashiwa

Reply:
It's the same event, we're buying the food at the Kashiwa site. It is very easy to access from central Tokyo, so splitting the community didn't make sense. We might host in Tokyo next year.

Reply:
Kashiwa is a suburb with a campus of Tokyo University, for the record.